libraries/Microsoft.Bot.Connector/Authentication/EndorsementsRetriever.cs

```diff
@@ -56,12 +56,12 @@ public EndorsementsRetriever(HttpClient httpClient)
         /// </summary>
         /// <param name="address">Address of the discovery document.</param>
         /// <param name="retriever">The document retriever to use to read the discovery document.</param>
-        /// <param name="cancellationToken">A cancellation token that can be used by other objects
+        /// <param name="cancel">A cancellation token that can be used by other objects
```

Comment:
Shouldn't we update the interface to call the parameter `cancellationToken` instead? Since it's the most used name across the libraries.

Reply:
Nevermind, it's an external interface. Sorry.